tests/test_text_onnx_embeddings.py

```diff
@@ -45,16 +45,16 @@
         [-0.0332, -0.0509, 0.0287, -0.0043, -0.0077]
     ),
     "jinaai/jina-embeddings-v2-base-de": np.array(
-      [-0.0085, 0.0417, 0.0342, 0.0309, -0.0149]
+        [-0.0085, 0.0417, 0.0342, 0.0309, -0.0149]
     ),
     "nomic-ai/nomic-embed-text-v1": np.array(
-        [0.0061, 0.0103, -0.0296, -0.0242, -0.0170]
+        [0.3708 ,  0.2031, -0.3406, -0.2114, -0.3230]
     ),
     "nomic-ai/nomic-embed-text-v1.5": np.array(
-        [-1.6531514e-02, 8.5380634e-05, -1.8171231e-01, -3.9333291e-03, 1.2763254e-02]
+        [-0.15407836, -0.03053198, -3.9138033, 0.1910364, 0.13224715]
```

Comment:
Were these embeddings obtained differently from `nomic-ai/nomic-embed-text-v1` ?

Why do they have more digits after the point?